docs/user-guide/pods.md

```diff
@@ -110,6 +110,15 @@ Pod is exposed as a primitive in order to facilitate:
 
 The current best practice for pets is to create a replication controller with `replicas` equal to `1` and a corresponding service. If you find this cumbersome, please comment on [issue #260](http://issue.k8s.io/260).
 
+## Privileges mode for pod containers
+
+From kubernetes v1.1, any container in a pod can enable privileged mode, using the `privileged` flag on the `SecurityContext` of the container spec. This is useful for containers that want to use linux capabilities like manipulating the network stack and accessing devices. Processes within the container get almost the same privileges that are available to processes outside a container. With privileged mode, it should be easier to write network and volume plugins as seperate pods that don't need to be compiled into the kubelet.
+
+If the master is running kubernetes v1.1 or higher, and the nodes are running a version lower than v1.1, then new privileged pods will not be launched. They will be pending state.
+
+If the master is running a version lower than v1.1, then privileged pods cannot be created.
```

Comment:
We already have this. We show an error:
The Pod "FooPodName" is invalid.
spec.containers[0].securityContext.privileged: forbidden '<*>(0xc20b222db0)true'